buildspecs/linux_ppc-64_cmprssptrs_le.spec

```diff
@@ -56,10 +56,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<property name="graph_make_parallel_arg" value="-j `numberOfCPUs`"/>
 		<property name="graph_req.arch0" value="arch:ppcle"/>
 		<property name="graph_req.arch1" value=""/>
-		<property name="graph_req.aux0" value="{$common.req.build.cuda$}"/>
+		<property name="graph_req.aux0" value="docker"/>
 		<property name="graph_req.aux1" value=""/>
-		<property name="graph_req.build" value="{$common.req.build.java9$}"/>
-		<property name="graph_req.build2" value="{$common.req.build.java8$}"/>
+		<property name="graph_req.build" value=""/>
+		<property name="graph_req.build2" value=""/>
```

Comment:
I don't think we should be removing the build capability here. We still use it for the other docker spec (xlinux). Without it, the job capabilities are fairly generic.

Reply:
Done

Reply:
Right, but it is a bit risky and out of the normal to not use the `build:` label for a compile job. It allows us to easily see if a machine is used for compiles. We still use it for the xlinux specs so I would prefer we keep it.